C-Plus-Plus/ds/SLL_Operations_Usnig_Unique_Key.cpp

```diff
@@ -0,0 +1,314 @@
+/*
+This is singly linked list implementation in which a unique key is alloted while entering the new node to the linked list, and you cannot give same key to the different node. In this code you can give duplicate data still it work fine.
```

Comment:
Modularize this and make the lines shorter and more readable

Reply:
- [x] Modularized the description and made the lines shorter and more readable.